app/Controllers/indexController.php

```diff
@@ -218,7 +218,7 @@ public function opmlAction() {
 				break;
 			case 'f':
 				// We most likely already have the feed object in cache
-				$feed = FreshRSS_CategoryDAO::findFeed($categories, $id);
+				$feed = FreshRSS_CategoryDAO::findFeed($categories, (int)$id);
```

Comment:
I have changed to cast higher up: https://github.com/FreshRSS/FreshRSS/pull/5064/commits/727ba1331c014cd6e1a69dbba9594cc7779e2aaa